openedx/core/djangoapps/discussions/views.py

```diff
@@ -69,6 +74,78 @@ def update(self, instance: LtiConfiguration, validated_data: dict) -> LtiConfigu
         return instance
 
 
+class LegacySettingsSerializer(serializers.BaseSerializer):
+    """
+    Serialize legacy discussions settings
+    """
+    class Meta:
+        fields = [
+            'allow_anonymous',
+            'allow_anonymous_to_peers',
+            'discussion_blackouts',
+            'discussion_link',
+            'discussion_sort_alpha',
+            'discussion_topics',
```

Comment:
I'd like to understand what goes into `discussion_topics` - in the frontend we have two checkboxes for "General" and "Questions for TAs" - in devstack for me, I saw some data about "General" come back in this field when I was testing the API. 

My question is whether or not `discussion_topics` is a dynamic list?  Can other things beside "General" and (possibly?) "Questions for TAs"  be added to it?  Do we need to account for that in the UI by making a dynamic list of checkboxes, rather than two hard-coded ones?

It looked like it had a label ("General") and a UUID-like ID; I'm not 100% sure how I can use that.  If I want to check or uncheck General, what do I send to the backend to indicate the change?  Membership in `discussion_topics` won't work, since that means if something's not there, I won't know to display it in the UI!  

Anyway, wondering if we can figure out how this thing works.